### DISCUSSION TITLE
Update repository url after Transfer ownership

### DISCUSSION THREAD

**Discussion by ihorkozar** [codemagic-ci-cd/codemagic-docs#1871] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I migrate from this repo https://github.com/ihorkozar/metrics to this https://github.com/Anadea/metrics
Codemagic app id: https://codemagic.io/app/62f3a33f033f9cf037f16b41

### Expected results

The documentation says the following:

"If the repository is added via GitHub, Bitbucket or GitLab integration, click the Update repository URL button. Codemagic will attempt to automatically update the URL based on the repository ID."

### Actual results

If i click on Update repository URL button, it showing me this message:
Cannot access repositories Not Found

**Comment by kulakarla** [comment]
Hello @ihorkozar 

The error makes sense since you cannot connect another users private repository with app.

Assuming you are using a personal account (not in any team project in Codemagic),  this @ihorkozar Github account is connected to it. If you want to use the new transffered repo at @Anadea, disconnect your current Github account and connect @Anadea with your Codemagic account. Then the update should work.

If you did transfer the repo to an organization however, which again I assume is the case here, then do the following:
1. Please make sure you are a member of the Anadea organization
2. Under Personal Account / Team settings (whichever you are actually using), go to Integrations -> Github -> Manage Integartion -> Configure.
3. There, connect the organization with Codemagic.
4. Now, when you are click "Update repository", it should connect to the migrated repository.

More info on Github integration with personal accounts, organizations can be found [here](https://docs.codemagic.io/getting-started/adding-apps/)

Hope this was helpful.

**Comment by kulakarla** [comment]
I am tentatively closing this thread due to inactivity.

**Reply by Adeiza-Dev** [reply]
I am having a similar issue. I transfer my repo to a different organization and now I can't update the URL on code magic (keep getting a 404) and it hasn't updated by itself

**Reply by helinanever** [reply]
Hi @Adeiza-Dev, please open a new issue so our support team can properly address your request. If you're on a paid plan, I recommend that you use the chat widget to reach out to support.
